bedrock/firefox/templates/firefox/nightly_firstrun.html

```diff
@@ -29,59 +31,61 @@
 {% block global_nav %}{% endblock %}
 
 {% block content %}
-
 <main role="main">
-  <header id="firstrun-header">
-    <h1>{{ _('Thank you for using Firefox Nightly') }}</h1>
-    <h2>{{ _('Choose an area to get involved below and help make Firefox better for users everywhere') }}</h2>
-  </header>
-  <div id="nightly-box">
-    <section class="blue-box test">
-      <header class="title">
-        <h4>{{ _('Test') }}</h4>
-      </header>
-      <div class="content">
-        <p class="description">
-          {{ _('Find and file bugs and generally make sure things work as they should.') }}
-        </p>
-        <a rel="external" data-element-type="button" href="{{ _('https://quality.mozilla.org/get-involved/') }}" class="button">{{ _('Start testing') }}</a>
-      </div>
-    </section>
-    <section class="blue-box code">
-      <header class="title">
-        <h4>{{ _('Code') }}</h4>
-      </header>
-      <div class="content">
-        <p class="description">
-          {{ _('File bugs and work on the building blocks of the Firefox browser.') }}
-        </p>
-        <a rel="external" data-element-type="button" href="{{ _('https://developer.mozilla.org/docs/Introduction') }}" class="button">{{ _('Start coding') }}</a>
-      </div>
-    </section>
-    <section class="blue-box localize">
-      <header class="title">
-        <h4>{{ _('Localize') }}</h4>
-      </header>
-      <div class="content">
-        <p class="description">
-          {{ _('Make Firefox available (and better) in more languages around the world.') }}
-        </p>
-        <a rel="external" data-element-type="button" href="{{ _('https://wiki.mozilla.org/L10n:Contribute') }}" class="button">{{ _('Start localizing') }}</a>
-      </div>
-    </section>
+  {{ hero(
+    title=_('Thank you for using Firefox Nightly'),
+    desc=_('Choose an area to get involved below and help make Firefox better for users everywhere'),
```

Comment:
This text is easy to miss grouped with the nightly logo and blurb. I think it makes more sense as a heading for the purple "contribute" section than as part of the hero component.

Not a blocker :)

Reply:
Good suggestion, I'll make it so 👍